app/(pages)/filmer/page.tsx

```diff
@@ -0,0 +1,10 @@
+import MovieList from '../../components/filmer/MovieList';
```

Comment:
We have decided to not use a components folder but put the components in the ui folder